star-dash/star-dash/Persistence/Database.swift

```diff
@@ -0,0 +1,197 @@
+//
+//  AppDatabase.swift
+//  star-dash
+//
+//  Created by Lau Rui han on 20/3/24.
+//
+
+import Foundation
+import os.log
+import SQLite
+
+struct Database {
+    static let DIR_DB = "StarDashDB"
+    static let DB_NAME = "stardash.sqlite3"
+    private let levelTable = Table("level")
+    private let entityTable = Table("entity")
+    private var db: Connection?
+    
+    init() {
```

Comment:
should we use a singleton design pattern for `Database`?

Reply:
I previously got marked down for singleton design pattern so not sure about this